### ISSUE TITLE
Unable to replicate results

### ISSUE THREAD

**Issue by RobrechtM** [cmusphinx/g2p-seq2seq#173]
When I run the metrics of the [pretrained model](https://sourceforge.net/projects/cmusphinx/files/G2P%20Models/g2p-seq2seq-model-6.2-cmudict-nostress.tar.gz/download), using the [Phonetisaurus test split](https://sourceforge.net/projects/cmusphinx/files/G2P%20Models/phonetisaurus-cmudict-split.tar.gz/download) I get:

* PER: 3.91%
* WER: 20.20%

Which is what I expected.  
However, when I train my own model with the training data provided with Phonetisaurus, I am not able to replicate the results:

* PER: 5.62%
* WER: 25.53%

I used the same configuration: 3 layers, 256 units and trained until early stopping at 200k steps. Why am I getting this big performance difference?

**Comment by nshmyrev** [comment]
It has been quite some time ago and I'm sorry I am afraid the proper config is lost. Also T2T has some regressions over time, they overoptimize for translation config but g2p might be worse. I would try the following: play with learning rate and batch size, don't rely on early stop, train for more iterations. I might check this, but it will not be fast. 

Thank you for the report.

**Comment by RobrechtM** [comment]
Thanks for your clarification.  
According to Tensorboard, the models could indeed be trained a bit longer. I'll look into it if I have the time.

**Comment by RobrechtM** [comment]
OK, after a lot of experimenting I finally managed to receive similar results:

- PER 4.87% 
- WER 20.42%

**Comment by nshmyrev** [comment]
Cool, please provide us the details of your training.
